ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1007,4 +1113,29 @@ FCU_US_MTV_1950_3:
   - FCU_DFSS_DSPRTC_DXDSPRTC
   - INCOMPLETE
   uses:
-  - cooling_stage_run_count
\ No newline at end of file
+  - cooling_stage_run_count
+
+FCU_RHC_DFVSC_RTC:
+  description: "CRAC with return air temp/humidity control and discharge/cooling fan monitor."
+  implements:
+  - FCU
+  - RHC
+  - DFVSC
+  - RTC
+  uses:
+  - discharge_fan_speed_percentage_sensor
+  - discharge_fan_run_status
```

Comment:
DFSS

Reply:
As discussed, this comment could be disregarded.